hoodie-utilities/src/main/java/com/uber/hoodie/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -22,7 +22,6 @@
 import static com.uber.hoodie.utilities.schema.RowBasedSchemaProvider.HOODIE_RECORD_STRUCT_NAME;
```

Comment:
@abhioncbr : Its better to revert code formatting here and just keep the  non-formatting changes. That way, its easier for others to merge their PR (which is touching this file).